server/src/main/java/io/unitycatalog/server/utils/Constants.java

```diff
@@ -7,6 +7,7 @@ public class Constants {
   public static final String SCHEMA = "schema";
   public static final String TABLE = "table";
   public static final String FUNCTION = "function";
+  public static final String DEFAULT_METASTORE_NAME = "uc_oss_metastore";
```

Comment:
How did we decide this name?

Reply:
no reason tbh

Reply:
Should we have a better default here? And with the metastore id as suffix to uniquify it?

The API description also states the metastore name is set by the user. Should we let the user optionally specify the metastore name during server init?

@tdas Need your inputs for this one.

Reply:
Remove the constant.